.github/workflows/main.yml

```
@@ -28,4 +28,4 @@ jobs:
       with:
         java-version: 8
     - name: Run Maven Targets
-      run: mvn package jacoco:report spotbugs:spotbugs checkstyle:checkstyle javadoc:javadoc coveralls:report --batch-mode --show-version --activate-profiles coveralls -Dcoveralls.skip=$COVERALLS_SKIP -DrepoToken=${{ secrets.COVERALLS_REPO_TOKEN }}
+      run: mvn -Dtest=\!MutantSwarmExtensionTest package jacoco:report spotbugs:spotbugs checkstyle:checkstyle javadoc:javadoc coveralls:report --batch-mode --show-version --activate-profiles coveralls -Dcoveralls.skip=$COVERALLS_SKIP -DrepoToken=${{ secrets.COVERALLS_REPO_TOKEN }}
```

```suggestion
      run: mvn -Dtest=\!MutantSwarmExtensionTest package jacoco:report spotbugs:spotbugs checkstyle:checkstyle javadoc:javadoc coveralls:report --batch-mode --no-transfer-progress --show-version --activate-profiles coveralls -Dcoveralls.skip=$COVERALLS_SKIP -DrepoToken=${{ secrets.COVERALLS_REPO_TOKEN }}
```
Would you mind trying to add this flag? It's something new in Maven 3.6.1 that should make the log output less noisy, I'm just not sure which version of Maven is in use on GitHub. If it doesn't work we can just remove it again.